definitions/marts/npq_separation_prep/tad_npq_enrolment_v2.sqlx

```diff
@@ -0,0 +1,90 @@
+config {
+    schema: "TAD",
+    database: "ecf-bq",
+    name: "tad_npq_enrolment_v2",
+    type: "table",
+    assertions: {
+        uniqueKey: ["application_id"]
+    },
+    bigquery: {
+        partitionBy: "DATE(application_created_at)",
+        clusterBy: ["cohort", "provider_name", "course_name", "application_status"]
+    },
+    description: "This table is designed around NPQ Applications with each application constituting a single record. Participant details have been joined into the application record.",
+    columns: {
+        application_id: "This is the unique id for an NPQ application. We join this field on the application_id field in the declarations table to identify declarations paired with this application",
+        teacher_profile_trn: "This TRN is sourced through user profiles which are dependent on the application having been accepted by a Lead Provider. If the application has been made but has yet to be accepted the TRN field will be blank despite a valid TRN verified field. ",
+        user_profile_id: "This User ID is sourced through user profiles which are dependent on the application having been accepted by a Lead Provider. If the application has been made but has yet to be accepted the User ID field will be blank.",
+        ecf_user_id: "This User ID is sourced through user profiles which are dependent on the application having been accepted by a Lead Provider. If the application has been made but has yet to be accepted the ECF User ID field will be blank. This field can be used to join on ECF Teacher Profiles should they have an ECF profile or were present in NPQ data prior to 28/11/2024.",

```

Comment:
I think this description is immediately too similar to the description above it. I would lead with 'This is the user_id that matches the user_ids created pre-separation in the ecf teacher profiles. (insert bit about 'This field can be used...') and then end with 'This user id is sourced ....'.